cromwell-helm/templates/cromwell.yaml

```diff
@@ -23,7 +23,7 @@ spec:
             claimName: {{ include "app.fullname" . }}-pvc
       containers:
         - name: {{ .Chart.Name }}-container
-          image: broadinstitute/cromwell:68-8e12ab5
+          image: broadinstitute/cromwell:69-ab51713-SNAP # TODO: Update this to official version once Cromwell 71 is released
```

Comment:
Not sure why, but for some reason when Katrina built and pushed docker image from her local branch, the hash produced was this instead of `71-<hash>-SNAP`